apps/application-system/api/src/app/modules/application/files/file.service.spec.ts

```diff
@@ -25,75 +25,70 @@ describe('FileService', () => {
 
   const applicationId = '1111-2222-3333-4444'
 
-  const parentA = {
-    nationalId: '0113215029',
-    ssn: '0113215029',
-    fullName: 'Test name',
-    phoneNumber: '111-2222',
-    email: 'email@email.is',
-  }
-
   const parentB = {
     id: 'id',
-    name: 'parent b',
-    ssn: '0022993322',
-    postalCode: '101',
-    address: 'Borgartún',
-    city: 'Reykjavík',
+    fullName: 'parent b',
+    nationalId: '0022993322',
+    address: {
+      postalCode: '101',
+      streetName: 'Borgartún',
+      city: 'Reykjavík',
+    },
+  }
+
+  const parentBWithContactInfo = {
+    ...parentB,
     phoneNumber: '222-1111',
     email: 'email2@email2.is',
   }
 
   const child = {
-    id: 'id',
-    name: 'child',
-    ssn: '123456-7890',
-    postalCode: '101',
-    address: 'Borgartún',
-    city: 'Reykjavík',
-    phoneNumber: '222-3333',
-    email: 'email3@email3.is',
+    fullName: 'child',
+    nationalId: '123456-7890',
```

Comment:
Should we remove the dash in the nationalId here, since we don't have it in other fake nationalId's in these tests?

Reply:
Sure, I'm pretty sure this is copied from older code though, thought there was some reason 😃 
```suggestion
    nationalId: '1234567890',
```